app/components/big-text.js

```diff
@@ -23,7 +23,12 @@ export default Component.extend({
     }
   }),
   cleanText: computed('text', function(){
-    const text = this.get('text') || '';
+    let text = this.get('text') || '';
+
+    if ((typeof text === 'object' && 'toString' in text || typeof text === 'number')){
```

Comment:
Woot!  Man I googled that crap forever yesterday.  I couldn't figure out how to look into the prototype chain and see what kind of Object something was.  
Thanks.